patches/lustre-cray-2.12.spec.patch

```diff
@@ -0,0 +1,19 @@
+--- lustre.spec.orig	2021-04-22 21:21:57.902995426 +0000
```

Comment:
When is this patch used? By which script?

Reply:
Updated description with explanations. Please check.

There is no script, one has to do this manually. Currently, needed for ARM64 nodes at SAGE, but can be used for any nodes or deployments.